mlflow/entities/trace_info.py

```diff
@@ -54,7 +54,7 @@ def to_proto(self):
         for key, value in self.tags.items():
             tag = ProtoTraceRequestMetadata()
             tag.key = key
-            tag.value = value
+            tag.value = str(value)
```

Comment:
IMO this conversion should be done upstream in the client layer. This is how it's done for run tags.



```suggestion
            tag.value = value
```

Reply:
makes sense!